v2/utils/utils.tsx

```diff
@@ -89,3 +89,55 @@ export const debounce = (fxn: () => any, ms: number) => {
         }, ms);
     };
 };
+
+export function formatDuration(seconds: number, sigfigs = 1) {
```

Comment:
I think there were tests?

Reply:
I couldn't find any tests for this, and I don't believe we have Jest set up in Argo UI. I could do this but I think it would be better in a separate PR